src/OpenTelemetry.AutoInstrumentation/Instrumentations/MongoDb/MongoClientIntegration.cs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 // </copyright>
 
-#if NETCOREAPP3_1_OR_GREATER
+#if NET6_0_OR_GREATER
```

Comment:
Maybe it would be easier for us if we use `#if !NETFRAMEWORK`?

Reply:
Not this PR.